packages/mask/src/plugins/RedPacket/SNSAdaptor/SelectNftTokenDialog.tsx

```diff
@@ -758,11 +754,10 @@ function NFTCard(props: NFTCardProps) {
                     loadingFailImage: classes.loadingFailImage,
                     iframe: classes.iframe,
                 }}
-                setERC721TokenName={setName}
             />
             <div className={classes.selectWrapperNftNameWrapper}>
                 <Typography className={classes.selectWrapperNftName} color="textSecondary">
-                    {name}
+                    {'#' + token.tokenId}
```

Comment:
We should not suppose that any NFT token uses incremental numbers as token IDs by default. Instead, some NFT tokens use hash strings as token id.

Reply:
As I know that hash string is hex number.

Reply:
That could be a long long string. Like this one: https://opensea.io/assets/0x699aa5e6eab40bcda7398ec7e5d585ca5d656a13/60212408056462910084986498866292153316932473047591013797793482224010201900028

Reply:
It needs to be formated just like addresses.

Reply:
Yeah, I will discuss this to PM.

Reply:
<img width="255" alt="image" src="https://user-images.githubusercontent.com/63733714/156156751-0a2b5fa6-ba9f-429d-bba9-5dd8cebcb00a.png">